### PR TITLE
Feature/calendar carousel

### PR DESCRIPTION
**Pull request by bml2238**
image
this also moves the milestones to be below the event carousel to match the mockups

### CODE REVIEW

File: backend/src/main/java/com/senior/project/backend/Activity/EventHandler.java [hunk 1]
```diff
@@ -19,4 +19,10 @@ public EventHandler(EventService eventService){
     public Mono<ServerResponse> all(ServerRequest serverRequest) {
         return ServerResponse.ok().body(this.eventService.all(), Event.class );
     }
+
+    public Mono<ServerResponse> dashboard(ServerRequest serverRequest) {
+        serverRequest.queryParam("studentID");
```
Comment (jts7382) [inline backend/src/main/java/com/senior/project/backend/Activity/EventHandler.java:24]: again we should get the student id from the auth token. We can handle that later

File: frontend/crd/src/app/dashboard/events/events.component.ts [hunk 2]
```diff
@@ -9,15 +9,46 @@ import {Event} from "../../../domain/Event";
...
 
   ngOnInit() {
-    this.eventService.getEvents().subscribe((events: Event[]) => {
+    //const isMobile = navigator.userAgent; //only display one event per page on mobile
+    //placeholder studentID
+    this.eventService.getDashboardEvents("student", 1).subscribe((events: Event[]) => {
       this.events = events;
+      console.log("events")
+      console.log(events);
+      console.log(this.slides);
```
Comment (jts7382) [inline frontend/crd/src/app/dashboard/events/events.component.ts:31]: please remove the logs

**Review by jts7382 — CHANGES_REQUESTED** [review]

File: frontend/crd/src/app/dashboard/events/events.component.html [hunk 3]
```diff
@@ -4,15 +4,18 @@ <h2>
...
-      <div>&nbsp;</div>
-    </ng-container>
-  </mat-card-content>
+  <carousel [itemsPerSlide]="itemsPerSlide"
+            [singleSlideOffset]="true"
+            [noWrap]="!noWrap"
+            [interval]="0"
+            [startFromIndex]="0">
+    <slide *ngFor="let slide of slides; let index=index">
+      <img [src]="slide.img" alt="event image" style="display: block; width: 100%; padding: 1%; background-color: darkslategrey;">
```
Comment (jts7382) [inline frontend/crd/src/app/dashboard/events/events.component.html:13]: You should do this with the Oswego logo component. just add an input and set this styling conditionally in the less

**Review by BasketOfEggs — APPROVED** [review]

**Comment by BasketOfEggs** [comment]
Approved for when Justin's suggestions are implemented

File: frontend/crd/src/app/dashboard/events/events.component.html (hunk 3 above)
Comment (bml2238) [reply frontend/crd/src/app/dashboard/events/events.component.html:13]: set the background image as the oswego logo component?

Comment (bml2238) [reply frontend/crd/src/app/dashboard/events/events.component.html:14]: leaving this as a placeholder

**Review by rcb2957 — APPROVED** [review]
Once the merges are fixed.  I approve the pull request.

**Review by jts7382 — APPROVED** [review]

**Pull request merged**